helm-charts/strimzi-kafka-operator/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+appVersion: "0.1.0"
+description: "Strimzi: Kafka as a Service"
+name: strimzi-kafka-operator
+version: 0.1.0
```

Comment:
Is this the chart version or should be the "software" version (in this case Strimzi) ? Should they be aligned ? Just asking ...

Reply:
`appVersion` is the "software" version.  `version` is the version of the actual chart.  

Often the charts in the kubeapps repo are managed by different people from the upstream project.  This is because the upstream project committers may not be concerned with how their software is deployed so somebody volunteers to do this on their behalf.  This could lead to an inefficient release process whenever you want to publish a new version of the chart, because the chart on kubeapps may significantly lag the latest release of the software.  IMO since Strimzi is a K8s specific project it makes sense to integrate the chart release into the overall release process.  A chart artifact could even be hosted by you immediately on a github pages site (a strimzi chart repo, basically).  Once the chart gets approved by kubeapps they have processes that let chart authors fast track new versions of the chart based on the github user details in the `OWNERS` file.  This essentially allows chart authors to create a PR that gets automatically merged once an owner approves it.  See this documentation on [Owning and Maintaining a Chart](https://github.com/kubernetes/charts#owning-and-maintaining-a-chart) for more details.  I think these steps could be added to your release process too after initial approval of the chart.